xml/System.Text.Json.Serialization/JsonConverter.xml

```diff
@@ -37,11 +37,11 @@
         <Parameter Name="typeToConvert" Type="System.Type" />
       </Parameters>
       <Docs>
-        <param name="typeToConvert">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="typeToConvert">The type to compare against.</param>
```

Comment:
nit:
```suggestion
        <param name="typeToConvert">The type of the object to check whether it can be converted by this converter instance.</param>
```